src/app/util/util.service.ts

```diff
@@ -53,4 +62,104 @@ export class UtilService {
   public objectsEqual(source: any, target: any): boolean {
     return deepEqual(source, target);
   }
+
+  // This Block works and could be used to persist the already rendered Blob element as a Data URI in the backend!
+  // At the moment, the LaTeX source code is persisted in the backend
+
+  // public getBlobFromDataUri(dataURI: string): Blob {
+  //   // convert base64 to raw binary data held in a string
+  //   // doesn't handle URLEncoded DataURIs - see SO answer #6850276 for code that does this
+  //   const byteString = atob(dataURI.split(',')[1]);
+  //
+  //   // separate out the mime component
+  //   const mimeString = dataURI.split(',')[0].split(':')[1].split(';')[0];
+  //
+  //   // write the bytes of the string to an ArrayBuffer
+  //   const ab = new ArrayBuffer(byteString.length);
+  //   const ia = new Uint8Array(ab);
+  //   for (let i = 0; i < byteString.length; i++) {
+  //     ia[i] = byteString.charCodeAt(i);
+  //   }
+  //
+  //   // write the ArrayBuffer to a blob
+  //   return new Blob([ab], { type: mimeString });
+  // }
+  //
+  // public getDataUriFromBlob(blob: Blob): string {
+  //   const fileReader = new FileReader();
+  //   fileReader.readAsDataURL(blob);
+  //   let dataUri: string;
+  //   fileReader.onload = () => {
+  //     dataUri = fileReader.result.toString();
+  //   };
+  //   return dataUri;
+  // }
```

Comment:
I would add a 'TODO' notice here, to clarify that this code could be used.

Reply:
done